### DISCUSSION TITLE
Error when submit NFT minting tx

### DISCUSSION THREAD

**Discussion by reitcircles** [Python-Cardano/pycardano#164] [General]
When submitting tx for the NFT minting of this example: https://github.com/Python-Cardano/pycardano/blob/main/examples/native_token.py

`blockfrost.utils.ApiError: {'error': 'Bad Request', 'message': '"transaction read error RawCborDecodeError [DecoderErrorDeserialiseFailure \\"Byron Tx\\" (DeserialiseFailure 1 \\"Size mismatch when decoding TxAux.\\\\nExpected 2, but found 4.\\"),DecoderErrorDeserialiseFailure \\"Shelley Tx\\" (DeserialiseFailure 1 \\"Size mismatch when decoding \\\\nRecord RecD.\\\\nExpected 4, but found 3.\\"),DecoderErrorDeserialiseFailure \\"Shelley Tx\\" (DeserialiseFailure 1 \\"Size mismatch when decoding \\\\nRecord RecD.\\\\nExpected 4, but found 3.\\"),DecoderErrorDeserialiseFailure \\"Shelley Tx\\" (DeserialiseFailure 1 \\"Size mismatch when decoding \\\\nRecord RecD.\\\\nExpected 4, but found 3.\\"),DecoderErrorDeserialiseFailure \\"Shelley Tx\\" (DeserialiseFailure 43 \\"expected string\\"),DecoderErrorDeserialiseFailure \\"Shelley Tx\\" (DeserialiseFailure 43 \\"expected string\\")]"', 'status_code': 400}`

Does anyone have a clue how to solve this ?

**Comment by nielstron** [comment]
This looks a lot like the issue discussed here and actually an issue of cardano-submit-api and not pycardano https://github.com/ImperatorLang/eopsin/discussions/46

**Comment by cffls** [comment]
This is the same issue as discussed [here](https://github.com/Python-Cardano/pycardano/issues/165).
